src/MLCompute/MLHelpers.cs

```diff
@@ -150,8 +217,14 @@ public static string GetDebugDescription (this MLCLstmResultMode self)
 		}
 	}
 
-	[TV (14,5)][Mac (11,3)][iOS (14,5)]
+#if NET
+	[SupportedOSPlatform ("ios14.0")]
```

Comment:
Why not 14.5, 11.3 and 14.5?

Reply:
that's a mistake that the test caught, fixed